var/spack/repos/builtin/packages/qemacs/package.py

```diff
@@ -24,18 +25,17 @@ class Qemacs(MakefilePackage):
 
     depends_on("which", type="build")
 
-    variant("docs", default=False, description="Build documentation")
+    variant("doc", default=True, description="Install documentation")
     variant("plugins", default=False, description="Enable plugin support")
     variant("X", default=False, description="Build with X11 support")
     variant("png", default=True, when="+X", description="Build with PNG support")
     variant("xshm", default=True, when="+X", description="Build with XShm support")
     variant("xv", default=True, when="+X", description="Build with X Video support")
 
-    conflicts("+docs", when="platform=freebsd")
     conflicts("+plugins", when="platform=freebsd")
     conflicts("+plugins", when="platform=darwin")
+    conflicts("+plugins", when="platform=windows")
```

Comment:
Is this essentially the following?
```suggestion
    requires("platform=linux", when="+plugins")
```

Reply:
Functionally mostly yes, but it's written as conflicts in the configure file